test/parallel/test-child-process-exec-maxBuffer.js

```diff
@@ -10,6 +10,29 @@ function runChecks(err, stdio, streamName, expected) {
   assert.deepStrictEqual(stdio[streamName], expected);
 }
 
+// default value
+{
+  const cmd = `"${process.execPath}" -e "console.log('a'.repeat(200 * 1024))"`;
+
+  cp.exec(cmd, common.mustCall((err) => {
+    assert.strictEqual(err.message, 'stdout maxBuffer length exceeded');
```

Comment:
If the default is infinity, this should not fail, right?

Reply:
Yes. I just checked default is not `200 * 1024`.
This test should fail #23027